src/tests/functional/plugin/conformance/subgraphs_dumper/CMakeLists.txt

```diff
@@ -51,5 +25,4 @@ addIeTargetTest(
         ADD_CPPLINT
```

Comment:
`DEPENDENCIES` is always empty here. I suppose we need to add `ov_frontends` here

Reply:
should we just depend on `ov_frontends` target which already depends on all frontends? It's more scalable way